recipe/meta.yaml

```diff
@@ -30,7 +29,8 @@ test:
   imports:
     - conda_libmamba_solver
   commands:
-    - CONDA_SOLVER_LOGIC=libmamba conda create -n test --dry-run scipy
+    - CONDA_SOLVER_LOGIC=libmamba conda create -n test --dry-run scipy  # [not win]
+    - conda create --experimental-solver libmamba -n test --dry-run scipy  # [win]
```

Comment:
I think we could even set it this way, to test --experimental-solver on all archs:
```
    - conda create --experimental-solver libmamba -n test --dry-run scipy
    - CONDA_SOLVER_LOGIC=libmamba conda create -n test --dry-run scipy  # [not win]
```